examples/metrics/src/main/java/com/google/cloud/opentelemetry/example/metrics/MetricsExporterExample.java

```diff
@@ -18,17 +18,23 @@
 import static java.util.Collections.singleton;
 
 import com.google.cloud.opentelemetry.metric.MetricExporter;
-import io.opentelemetry.api.metrics.GlobalMetricsProvider;
-import io.opentelemetry.api.metrics.LongUpDownCounter;
+import io.opentelemetry.api.metrics.LongCounter;
 import io.opentelemetry.api.metrics.Meter;
 import io.opentelemetry.sdk.metrics.SdkMeterProvider;
 import io.opentelemetry.sdk.metrics.export.IntervalMetricReader;
 import java.io.IOException;
 import java.util.Random;
+import org.apache.log4j.BasicConfigurator;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class MetricsExporterExample {
+  private static final Logger logger = LoggerFactory.getLogger(MetricsExporterExample.class);
+
+  private static SdkMeterProvider METER_PROVIDER =
```

Comment:
nit: Ideally these changes should be reviewed and merged separately, as they are not related to adding support for GKE resource.

But given that they are fixing issues with the example it might be okay to merge now and improve further later if needed.